packages/ui-components/src/components/windowedlist.ts

```diff
@@ -1256,9 +1333,16 @@ export class WindowedList<
   }
 
   /**
-   * Handle viewport content (e.g. widgets) resize.
+   * Handle viewport area resize.
+   */
+  private _onAreaResize(_entries: ResizeObserverEntry[]): void {
+    this._scrollBackToItemOnResize();
```

Comment:
Just checking my own understanding here. 

Is this line more for code symmetry and to cover edge cases than to cover a commonly occurring scenario? I mean, it's generally unlikely that an area resize will happen within 100 ms of the app scrolling to some item, right?

Reply:
No, it is actually quite common when user executes a cell which produces output and advances to the next cell. See:

![image](https://github.com/jupyterlab/jupyterlab/assets/5832902/ea005857-401e-4f6b-95cf-296cbb9a6713)

Reply:
Ah, for some reason I thought the area resize observer was observing the outer (scrollable) element (for e.g. window resize). Thanks for the clarification.